com.unity.formats.alembic/Editor/Importer/AlembicImporter.cs

```diff
@@ -203,6 +211,27 @@ public override void OnImportAsset(AssetImportContext ctx)
             firstImport = false;
         }
 
+        [InitializeOnLoadMethod]
+        static void InitializeEditorCallback()
+        {
+            EditorApplication.update += DirtyCustomDependencies; //
+        }
+
+        static string pipelineName = string.Empty;
+        static void DirtyCustomDependencies()
+        {
+            var newPipelineName = GraphicsSettings.currentRenderPipeline == null
+                ? ""
+                : GraphicsSettings.currentRenderPipeline.ToString();
```

Comment:
Use AssetDatabase.TryGetGUIDAndLocalFileIdentifier instead of the name and compute the hash from the guid and localId.
This will handle multiple pipelines with the same names and multiple pipelines declared in one configuration asset (yes, we don't do it usually, but that's a doable thing that needs to be taken into account here for determinism)